src/components/CreateGoalForm/DatePicker.tsx

```diff
@@ -55,4 +46,15 @@ const styles = StyleSheet.create({
     width: 205,
     marginTop: 2,
   },
+  dateIconStyle:{
+    position: 'absolute',
+    left: 0,
+    top: 4,
+    marginLeft: 0,
+  },
+  dateInputStyle:{
+    marginLeft: 36,
+    height: 30,
+    border: 'none',
+  }
 });
```

Comment:
can we move styling in different files?

Reply:
Please Review now I have moved css to separate files